modules/apps/transfer/types/ack_test.go

```diff
@@ -80,21 +79,3 @@ func (suite *TypesTestSuite) TestABCICodeDeterminism() {
 	suite.Require().Equal(hash, hashSameABCICode)
 	suite.Require().NotEqual(hash, hashDifferentABCICode)
 }
-
-// TestAcknowledgementError will verify that only a constant string and
-// ABCI error code are used in constructing the acknowledgement error string
-func (suite *TypesTestSuite) TestAcknowledgementError() {
```

Comment:
Should we keep one of these two `TestAcknowledgementError` tests and move it to `04-channel`?

Reply:
Yes, I think the `TestABCICodeDeterminism` test should also be moved to `04-channel`, its duplicated both above here and in ICA tests